Week 05/Yoana Borisova SR

```diff
@@ -0,0 +1,97 @@
+void Swap(int& num1, int& num2)
+{
+    int temp = num1;
+    num1 = num2;
+    num2 = temp;
+}
+
+int* getArr(int num)
```

Comment:
I am not quite sure what this was intended to do. The function is used in the context of printing the elements of an array. This function does pretty much nothing as is.